assignment2.txt

```diff
@@ -0,0 +1,11 @@
+Bettina Benitez
+
+1. You can combine two or more operators. However, you must include && or ||.
+2. Logical errors happen when the output of the code is unexpected while Syntax errors are grammatical errors in the code.
```

Comment:
Correct - it's important to also note that logical errors occur during runtime while syntax errors occur during compile time.